### PR TITLE
Removed jquery-ui

### LINKED ISSUE #3001: Change registration modal (login and about gui) to use bootstrap instead of jquery-ui
Change registration modal (login and about gui) to use bootstrap instead of jquery-ui

### PR DESCRIPTION
**Pull request by adad20**
Fixes #3001  

#### Short description of what this resolves:
Uses bootstrap instead of jquery-ui

### CODE REVIEW

**Comment by adad20** [comment]
@im-Amitto Can you review this and suggest if some changes are required.

**Comment by bradymiller** [comment]
hi @adad20 ,

Here's what the modals look like in current codebase:

image

image

<br>
Testing this PR doesn't show any modal and instead get a js console error with `TypeError: $(...).dialog is not a function`
image

**Comment by im-Amitto** [comment]
@adad20 `/interface/product_registration/product_registration_controller.js:47` replace that logic with a bootstrap modal as .dialog() belong to jquery-ui.

**Comment by adad20** [comment]
@bradymiller Is this modal automatically showed up or shall we create a button to open this modal?

**Comment by bradymiller** [comment]
hi @adad20 ,
The algorithm for when it shows is already worked out and shouldn't require any mods.
(it autoshows on first login, and will never show again after user clicks 'No Thanks')
To see how it works, check out the Development Demo links (use the ones without demo data) (there are 4 of those demos there):
https://www.open-emr.org/wiki/index.php/Development_Demo#Daily_Build_Development_Demos
thanks, -brady

**Comment by adad20** [comment]
@bradymiller Okay

**Comment by bradymiller** [comment]
Note the About gui is also using the same mechanism.

**Comment by im-Amitto** [comment]
@adad20 Let me know if you hit a wall :).

**Comment by adad20** [comment]
@im-Amitto yes. Are you active on slack? I want to ask some queries.

**Comment by im-Amitto** [comment]
@adad20 Yes, my account name is `Amit Meena`

**Comment by adad20** [comment]
@bradymiller I accidentally pressed on thanks button now the modal is not appearing. What can I do to reopen it?

**Comment by bradymiller** [comment]
hi @adad20 , Drop the entry in the product_registration sql table (there will only be one entry in it) and then it will come back.

**Comment by adad20** [comment]
@im-Amitto Can you have a look?

File: interface/login/login.php [hunk 1]
```diff
@@ -341,12 +341,28 @@ function transmit_form(element) {
                         <a href="../../acknowledge_license_cert.html" target="main"><?php echo xlt('Acknowledgments, Licensing and Certification'); ?></a>
                     </p>
                 </div>
-                <div class="product-registration-modal" style="display: none">
-                    <p class="context"><?php echo xlt("Register your installation with OEMR to receive important notifications, such as security fixes and new release announcements."); ?></p>
-                    <input placeholder="<?php echo xlt('email'); ?>" type="email" class="email w-100 text-body" />
-                    <p class="message font-italic"></p>
+                <div class="product-registration-modal modal fade w-50 ">
```
[the remark this answers is not in the document]
Comment (adad20) [reply interface/login/login.php]: @im-Amitto Can u link some resource I don't know about dlgopen()...

[the remark this answers is not in the document]
Comment (stu01509) [reply interface/login/login.php]: Hi @adad20 

I think, you can follow [library/dialog.js](https://github.com/openemr/openemr/blob/master/library/dialog.js) this file,
This file contains a lot of comments about dlgopen() function usage.

**Pull request closed without merging**